the_debuginator.h

```diff
@@ -1090,6 +1091,18 @@ DebuginatorItem* debuginator_create_array_item(TheDebuginator* debuginator,
 	item->leaf.description = description == NULL ? "" : description;
 	debuginator__adjust_num_visible_children(item->parent, 1);
 
+	if (description && !item->is_folder) {
+		const char* description = item->leaf.description;
```

Comment:
Not a fan of creating a new variable with the same name as one in an earlier scope. Maybe just use `item->leaf.description` inside the if?